iris/tests/find_toolbar/navigate_quick_find.py

```diff
@@ -14,7 +14,7 @@ def __init__(self):
         self.test_case_id = '127260'
         self.test_suite_id = '2085'
         self.locales = ['en-US']
-        self.blocked_by = 'issue 1628'
```

Comment:
We should leave 'issue_' as part of the test case ID to differentiate between Bugzilla and GitHub issues

Reply:
GitHub issues have been updated in order to differentiate them from the Bugzilla issues.

Thanks.